src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/management/PageTestsIT.java

```diff
@@ -0,0 +1,153 @@
+      .get("/organizations/{organizationId}/pages/{pageId}", organizationId, getOrganizationPageId(organizationId, 0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("slug", is("zeus"))
+      .body("parentId", nullValue())
+      .body("titles.size()", is(1))
+      .body("titles[0].language", is("fi"))
+      .body("titles[0].value", is("Zeus"))
+      .body("titles[0].type", nullValue());
+  }
+  
+  @Test
+  public void testListPages() {
+    // Zeus shoud be listed before abraham and bertha because it's menu_order is set to -100 
+
+    String organizationId = getOrganizationId(0);
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/pages", organizationId)
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("slug[0]", is("zeus"))
+      .body("slug[1]", is("abraham"))
+      .body("slug[2]", is("bertha"));
+  } 
+  
+  @Test
+  public void testListPagesByPath() {
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/pages?path=/bertha", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(1))
+      .body("id[0]", notNullValue())
+      .body("slug[0]", is("bertha")); 
+    
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/pages?path=/non-existing", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(0));
+  } 
+    
+  @Test
+  public void testOrganizationPagesNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationAnnouncecmentId = getOrganizationPageId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/pages/%s", organizationId, organizationAnnouncecmentId));
+    assertEquals(3, countApiList(String.format("/organizations/%s/pages", organizationId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/pages/%s", organizationId, malformedId));
+    }
+    
+    assertNotFound(String.format("/organizations/%s/pages/%s", incorrectOrganizationId, organizationAnnouncecmentId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/pages", incorrectOrganizationId)));
+  }
+  
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+    
+  private void createManagementSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "createManagementSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)